tests/commandLine.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	'basic functionality': browser => {
+		browser
+			.url('https://www.reddit.com/wiki/pages/')
+			.waitForElementVisible('#RESSettingsButton')
+			.keys([browser.keys['.']])
+			.assert.visible('#keyCommandLineWidget')
+			.keys([browser.keys.SHIFT, browser.keys.SLASH]) // "?"
+			.assert.containsText('#keyCommandForm', 'navigates to subreddit')
```

Comment:
```
erik [6:27 PM] I'll explicitly make it `en.reddit.com` so it's not i18n'd if that sounds good, since there's not much else to look for (classless <li>s, but they could be empty if it bugs out)
andytuba [6:29 PM] +1
```